src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -2096,7 +2193,50 @@ private void ValidateOrganizationUpgradeParameters(Models.StaticStore.Plan plan,
             upgrade.AdditionalSeats > plan.MaxAdditionalSeats.Value)
         {
             throw new BadRequestException($"Selected plan allows a maximum of " +
-                $"{plan.MaxAdditionalSeats.GetValueOrDefault(0)} additional users.");
+                                          $"{plan.MaxAdditionalSeats.GetValueOrDefault(0)} additional users.");
+        }
+    }
+
+    private static void ValidateSecretsManagerPlan(Models.StaticStore.Plan plan, OrganizationUpgrade upgrade)
```

Comment:
This method has some identical lines of code with `ValidatePasswordManagerPlan`, can we have a base method for both?

Reply:
resolved